css/src/atomics/table.scss

```diff
@@ -0,0 +1,108 @@
+/* stylelint-disable selector-no-qualifying-type */
```

Comment:
instead of disabling, we should fix the rules.

Reply:
I need some help with it cause I don't understand why I'm getting this particular error on these places:

```
.table {
  td,
  th {
    &.nowrap {
      white-space: nowrap;
    }

    &.is-narrow {
      width: $table-cell-narrow-width;
    }
  }
}
```

Reply:
qualifying selector type doesn't like the pattern `a.class-name`. It's hard to see because of the nesting, but it doesn't like `td.nowrap, th.is-narrow` etc.

Reply:
Looks like we'll probably be taking a lot of that stuff out, so perhaps we won't need to worry about it?

Reply:
I've got another lint error that I had to disable cause I don't get how to resolve it. Here is the structure:

```
.table {
	...

	th,
	td {
		display: table-cell;
		...

		&:not([align]) {
			text-align: $user-left;
		}
	}

	thead th {
		border-top: 0;
		border-bottom: 1px solid $border;
	}
}
```

Here is an error I'm getting:

```
Expected selector ".table thead th" to come before selector ".table th:not([align])"   no-descending-specificity
```

I tried to move `thead th { ... }` block before the `th, td { ... }`, but then I'm getting this error:

```
Expected selector ".table th" to come before selector ".table thead th"   no-descending-specificity
```

So it's a circle 😵

Reply:
```
	th:not([align]),
	td:not([align]) {
		text-align: $user-left;
	}
```